rudder-web/src/main/webapp/templates-hidden/server/server_details.html

```diff
@@ -37,18 +37,18 @@
 <detail:server>
 <div id="node_tabs" class="tabs">
   <div id="node_header" class="nodeheader">
-    <div class="nodeheadercontent ui-corner-top"> Node Details - <span id="node_name"></span>
+    <div class="nodeheadercontent ui-corner-top"> Node: <span id="node_name"></span>
     </div>
   </div>
 
 
     <ul id="NodeDetailsTabMenu">
-      <li><a href="#node_summary">Node summary</a></li>
+      <li><a href="#node_summary">Summary</a></li>
+      <li><a href="#node_reports">Compliance</a></li>
```

Comment:
This line is both a rename (Reports --> Compliance) and a re-order, so that the Compliance tab is displayed right at the beginning of Node details. This seems more logical than hiding at the end, since Rudder is all about compliance.